cmake/kconfig.cmake

```diff
@@ -39,6 +39,8 @@ set(ENV{SOC_DIR}   ${SOC_DIR})
 set(ENV{CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR})
 set(ENV{ARCH_DIR}   ${ARCH_DIR})
 set(ENV{GENERATED_DTS_BOARD_CONF} ${GENERATED_DTS_BOARD_CONF})
+set(ENV{DTS_POST_CPP} ${DTS_POST_CPP})
+set(ENV{DTS_ROOT_BINDINGS} ${DTS_ROOT_BINDINGS})
```

Comment:
Maybe these variables could this be called `BINDINGS_DIR` as well, for consistency.

Reply:
The issue is we'd end up with something like:

`set(ENV{BINDINGS_DIR} ${DTS_ROOT_BINDINGS})`

Reply:
Is `DTS_ROOT_BINDINGS` used elsewhere? Was thinking it could be renamed too.